clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -11253,6 +11253,9 @@ void LinkerWrapper::ConstructJob(Compilation &C, const JobAction &JA,
     if (IsUsingLTO && LTOMode == LTOK_Thin)
       CmdArgs.push_back(Args.MakeArgString("-sycl-thin-lto"));
 
+    if (Args.hasArg(options::OPT_fsycl_embed_ir))
+      CmdArgs.push_back(Args.MakeArgString("-sycl-embed-ir"));
+
```

Comment:
nit

```suggestion
    Args.AddLastArg(CmdArgs, options::OPT_fsycl_embed_ir);

```

Reply:
I don't think this will work.  The new option as implemented is `-sycl-embed-ir`

Reply:
oh sorry I missed that